libobs/util/platform-windows.c

```diff
@@ -975,6 +975,26 @@ bool is_64_bit_windows(void)
 #endif
 }
 
+bool is_arm64_windows(void)
+{
+#if defined(_M_ARM64) || defined(_M_ARM64EC)
+	return true;
+#else
+	USHORT _, nativeMachine;
+	auto result = IsWow64Process2(GetCurrentProcess(), &_, &nativeMachine);
+	return (result && (nativeMachine == IMAGE_FILE_MACHINE_ARM64));
+#endif
+}
+
+bool os_get_emulation_status()
+{
+#if defined(_M_ARM64) || defined(_M_ARM64EC)
+	return false;
+#else
+	return is_arm64_windows();
+#endif
+}
```

Comment:
@gxalpha Actually there is a difference. 

For `is_arm64_windows(void)`, if we want to know if we are running on ARM64 Windows, and we are ARM64 binary, then return true because only ARM64 Windows can run ARM64 binary. 

For `os_get_emulation_status()`, if we want to know if we are being emulated, and we are ARM64 binary, then return false because emulation only works one way as I said.